### PR TITLE
Create 2024-09-12.mdx

### PR DESCRIPTION
**Pull request by carstenjacobsen**
Adding meeting notes from Stellar Developer Meeting held at 09/12 2024, and adding carstenjacobsen as a contributor.

### CODE REVIEW

**Comment by anataliocs** [comment]
Hey @sreuland you reviewed my meeting notes PR, so I added you to this one, but if someone else should review let me know!

File: meeting-notes/2024-09-12.mdx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+tags: [developer]
+---
+
+<iframe src="https://drive.google.com/file/d/1raU1Z9EQBLCkkuUO9yyF8yD2l_3nwk4a/preview" width="640" height="480" allow="autoplay"></iframe>
+
+Agenda: [Discord thread](https://discord.com/channels/897514728459468821/900374272751591424/1282934024892973077)
+
+Developer Experience team member Nando Vieira introduced the CLI features alias and snapshot:
+
+1. alias
```
Comment (anataliocs) [inline meeting-notes/2024-09-12.mdx:13]:
```suggestion
1. Alias
```

File: meeting-notes/2024-09-12.mdx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+     1. Install of Hello World example for showcasing alias
+     2. Showed examples of how contract IDs are often passed as parameters in CLI commands like invoke (copying ID string or command substitution)
+     3. How to deploy a smart contract and create an alias
+     4. How to invoke the smart contract with the alias
+2. snapshot
```
Comment (anataliocs) [inline meeting-notes/2024-09-12.mdx:18]:
```suggestion
2. Snapshot 
```

**Review by anataliocs — CHANGES_REQUESTED** [review]
Reviewed the preview and it looks good:

image

@fnando Does this look good to you since you were the presenter? :)

**Review by anataliocs — APPROVED** [review]
LGTM 🚢 

@fnando Any requested changes from your end?  Otherwise, we'll just merge it in.

**Review by janewang — COMMENTED** [review]
LGTM

**Pull request merged**